test/graphql.matchingFund.nock.js

```diff
@@ -118,84 +176,90 @@ export default function() {
         'Server',
         'nginx',
         'Date',
-        'Thu, 28 Dec 2017 22:24:41 GMT',
+        'Fri, 01 Mar 2019 15:41:59 GMT',
         'Content-Type',
         'application/json',
         'Content-Length',
-        '781',
+        '785',
         'Connection',
-        'close',
+        'keep-alive',
         'Access-Control-Allow-Credentials',
         'true',
         'Access-Control-Allow-Methods',
         'GET, POST, HEAD, OPTIONS, DELETE',
         'Access-Control-Allow-Origin',
         '*',
         'Access-Control-Expose-Headers',
-        'X-Stripe-Privileged-Session-Required,stripe-manage-version,X-Stripe-External-Auth-Required',
+        'Request-Id, Stripe-Manage-Version, X-Stripe-External-Auth-Required, X-Stripe-Privileged-Session-Required',
         'Access-Control-Max-Age',
         '300',
         'Cache-Control',
         'no-cache, no-store',
         'Request-Id',
-        'req_1qqgZSHGoXNOOx',
+        'req_bRBIfM5O0cLdlB',
         'Stripe-Version',
         '2015-04-07',
         'Strict-Transport-Security',
         'max-age=31556926; includeSubDomains; preload',
       ],
     );
 
-  nock('http://data.fixer.io', { encodedQueryParams: true })
+  nock('http://data.fixer.io:80', { encodedQueryParams: true })
```

Comment:
Does this make sense? Which library is calling this?

Reply:
This is used in `server/lib/currency.js`, looks like some internal changed, but the semantic is the same.